custom/style.css

```diff
@@ -177,6 +177,27 @@ mfrac msup :not(:first-child) {
     text-align: center;
 }
 
+/* Improve contrast on bit-descrs borders */
+.ayu .bit-descrs {
+  --table-border-color: hsl(210, 25%, 30%);
+}
+
+.coal .bit-descrs {
+  --table-border-color: hsl(200, 7%, 30%);
+}
+
+.light .bit-descrs {
+  --table-border-color: hsl(0, 0%, 70%);
+}
+
+.navy .bit-descrs {
+  --table-border-color: hsl(226, 23%, 40%);
+}
+
+.rust .bit-descrs {
+  --table-border-color: hsl(60, 9%, 60%);
```

Comment:
Do all of those match the `--table-header-bg`?

If so, perhaps all of these could be collapsed into a single
```css
.bit-descrs {
  --table-border-color: var(--table-header-bg);
}
```
rule, which would be simpler and also track any upstream changes.

Reply:
haven't though about it, that might look even better, I'll try and update